usaspending_api/common/api_request_utils.py

```diff
@@ -253,6 +253,13 @@ def validate_post_request(self, request):
                                 raise InvalidParameterException("Invalid field, operation 'range_intersect' requires an array of length 2 for field")
                             if (not isinstance(filt['value'], list) or len(filt['value']) != 2) and 'value_format' not in filt:
                                 raise InvalidParameterException("Invalid value, operation 'range_intersect' requires an array value of length 2, or a single value with value_format set to a ranged format (such as fy)")
+                        if filt['operation'] == 'search':
```

Comment:
Took me a minute to puzzle these next few lines out; the upshot is that we're raising an error if any field in filt['field'] is non-string, whether filt['field'] is a list or a single value... is that right?  Maybe (non-urgently) a comment to that effect, or even pull it out to a function with an informative name like `raiseIfNonFieldPresent` or something...